lib/MIDI/RtMidi/ScorePlayer.pm

```diff
@@ -12,7 +12,9 @@ use File::Basename qw(fileparse);
 use MIDI::RtMidi::FFI::Device ();
 use MIDI::Util qw(dura_size get_microseconds score2events set_chan_patch ticks);
 use Path::Tiny qw(path);
-use Time::HiRes qw(time usleep);
+use Time::HiRes qw(time);
+use Future::IO ();
+use Future::AsyncAwait;
```

Comment:
I didn't add the exports explicitly to this `use` as it throws `Unrecognised import symbol async` - it's not a normal `Exporter`...